evalml/pipelines/components/transformers/preprocessing/time_series.py

```diff
@@ -27,7 +27,7 @@ def __init__(self, max_delay=2, delay_features=True, delay_target=True, gap=1,
                 when the target is collected. For example, if you are predicting the next time step's target, gap=1.
                 This is only needed because when gap=0, we need to be sure to start the lagging of the target variable
                 at 1.
-            random_state (int, np.random.RandomState): Seed for the random number generator. There is no randomness
+            random_state (int): Seed for the random number generator. There is no randomness
```

Comment:
Isn't there? lol, what does this mean?

Reply:
I believe this transformer performs the same regardless of the random seed fed to it? @freddyaboulton is this correct?

Reply:
Yes, the random_state arg was added to match the api of other transformers. Maybe "This transformer performs the same regardless of the random seed fed to it" or something like that is the better thing to say 😆

Reply:
I'll fix it for clarity